documentation/manual/src/main/asciidoc/modules/configuration.asciidoc

```diff
@@ -523,6 +504,77 @@ Add a `WEB-INF/jboss-deployment-structure.xml` in your archive with the followin
 More information about the descriptor can be found in the
 https://docs.jboss.org/author/display/WFLY10/Class+Loading+in+WildFly[WildFly documentation].
 
+===== Server provisioning via other build tools
+
+The provisioning tools have been developed mainly for Maven users, but there are options for users of other build tools too.
+
+Gradle users can use the link:https://plugins.gradle.org/plugin/org.wildfly.build.provision[org.wildfly.build.provision] plugin, available on the official Gradle plugin portal.
+
+Both the Maven and the Gradle plugins are just thin wrappers invoking a standalone provisioning library, so you could also integrate this library with any other JVM based tool.
+
+See also https://github.com/wildfly/wildfly-build-tools[WildFly provisioning build tools].
+
+[[hibernate-ogm-jboss-modules-feature-packs]]
+==== List of the Hibernate OGM WildFly/JBoss feature packs
+
+===== Core feature pack
+
+This is the base feature pack, extended by all Hibernate OGM dialect feature packs.
+It includes the main module:
+
+* _org.hibernate.ogm_, containing Hibernate OGM core module
+
+It includes Hibernate ORM {hibernate-orm-version} modules, required by Hibernate OGM {hibernate-ogm-version}.
+Several more modules are included but you should not need to refer to the other ones directly.
+
+The feature pack is published on the JBoss Nexus repository and Maven Central as
+link:$$https://repository.jboss.org/nexus/index.html#nexus-search;gav~org.hibernate.ogm~hibernate-ogm-featurepack-core~{hibernate-ogm-version}~~$$[org.hibernate.ogm:hibernate-ogm-featurepack-core:{hibernate-ogm-version}:zip].
+
+===== Infinispan Remote feature pack
+
+This is the Infinispan Remote dialect feature pack.
+It includes the main module:
+
+* _org.hibernate.ogm.infinispan-remote_, containing Hibernate OGM Infinispan Remote module
```

Comment:
It took me a while to understand that this is the module on WIldfly after you install the feature packs.
I think that for a user it would be more important to have the cooridnates of the feature pack first, possibly with the syntax: 
```
		<feature-pack
			groupId="org.hibernate.ogm"
			artifactId="hibernate-ogm-featurepack-infinispan-embedded"
			version="${project.version}" />

```

This way, they can copy and paste it in their provisioning file. Then, you can describe the rest. But maybe I'm missing something, let me know if what I'm saying doesn't make sense.

Reply:
I think we should also add the feature pack configuration line in the section about sa single datastore. So that the user can find it when looking at a specific datastore as well

Reply:
I copy this kind of description from Hibernate Search documentation: https://docs.jboss.org/hibernate/search/5.10/reference/en-US/html_single/#hibernatesearch-jboss-modules-feature-packs.